arbiter-core/src/lib.rs

```diff
@@ -1,3 +1,22 @@
+//! `arbiter-core` is designed to facilitate agent-based simulations of Ethereum
+//! smart contracts in a local environment.
+//!
+//! With a primary emphasis on ease of use and performance, it employs the [`revm`](https://crates.io/crates/revm) (Rust EVM) to provide a local execution environment that closely simulates the Ethereum blockchain but without associated overheads like networking latency.
+//!
+//! Key Features:
+//! - **Manager Interface**: The main user entry-point that offers management of
+//!   different environments and agents.
+//! - **Environment Handling**: Detailed setup and control mechanisms for
+//!   running the Ethereum-like blockchain environment.
+//! - **Middleware Implementation**: Customized middleware to reduce overhead
+//!   and provide optimal performance.
+//!
+//! For a detailed guide on getting started and best practices, check out [link
+//! to your guide or further documentation]. // TODO: Add in a link.
```

Comment:
Should we add this link for this todo?

Reply:
Yes we should, but I don't know where it will link to.

Reply:
okay let's leave for now until we know and deal with it post release. I will make an issue for it so we don't forget.